x-pack/plugins/beats_management/server/routes/beats/events.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { IRouter } from 'kibana/server';
+import { ReturnTypeBulkAction } from '../../../common/return_types';
+import { wrapRouteWithSecurity } from '../wrap_route_with_security';
+
+/*
+import Joi from 'joi';
+import { FrameworkRequest } from '../../lib/adapters/framework/adapter_types';
+import { CMServerLibs } from '../../lib/types';
+export const beatEventsRoute = (libs: CMServerLibs) => ({
+  method: 'POST',
+  path: '/api/beats/{beatId}/events',
+  config: {
+    validate: {
+      headers: Joi.object({
+        'kbn-beats-access-token': Joi.string().required(),
+      }).options({ allowUnknown: true }),
```

Comment:
This misleading `kbn-beats-access-token` token validation is actually doing nothing, as the object is not `.required()`. Also, the header is not used in the route handler, and neither the FTR test suite nor the client-side API are sending the header, which make me think this was just bad copypasta. 

All that to say: the fact that this validation is not present on the KP handler is not a mistake.